dist/amazon-cognito-identity.js

```diff
@@ -3656,6 +3657,11 @@ return /******/ (function(modules) { // webpackBootstrap
 	    } else {
 	      this.expires = 365;
 	    }
+	    if (data.secure) {
```

Comment:
This line should be `data.hasOwnProperty('secure')` because the `if(data.secure)` will never evaluate to true when `data.secure = false`.

Reply:
Done. I didn't realize at the time it was a boolean variable